test/common/schemaform/SaveFormLink.unit.spec.jsx

```diff
@@ -34,14 +34,24 @@ describe('Schemaform <SaveFormLink>', () => {
   //  prop requirements; they're only passed to LoginModal which we test elsewhere
   const saveInProgressForm = sinon.spy();
   const updateLoginSpy = sinon.spy();
-  it('should render login message when not logged in', () => {
+  it('should render save message when not logged in', () => {
```

Comment:
Interesting, this makes me wonder whether it might be helpful for the user to know that clicking on "Save and finish this application later" will prompt them to sign in. Could it be helpful to rephrase this as "Sign in to save and finish this application later"? @peggygannon @stacyhlinn (Not a blocking question.)

Reply:
I think the latter phrasing is a little ambiguous as to what happens when you click the link: are you just signing in or are you signing in and saving the application?

Reply:
Good point, if it seems helpful, perhaps "Sign in and save this application to finish it later" is another option. Whether it's helpful is still open to debate.

Reply:
@cehsu @jbalboni @stacyhlinn I'm wondering if we could set this link as a box, and mirror the SIP messaging treatment that is on form introduction pages. So copy would be in a shaded box with the following message:
You can save this form, and come back later to finish filling it out. [Sign in to save your form in progress]. 

This would also provide consistent messaging to the user throughout the forms.